.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,20 @@
 # Fill out the following while publishing a new pull request.
 
 * **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
-
+This commit is used to explain what is MVP and how the current project is designed to proceed.
```

Comment:
Update your pull request to match this template and revert this commit. The template is meant to be used by everyone while submitting a pull request.